README.md

```diff
@@ -1,5 +1,7 @@
 # PDF parser
 
+Forked and merged with https://github.com/smalot/pdfparser/pull/666
```

Comment:
Why this line?

Reply:
Sorry, I am using a fork with this fix and it was my internal information. I have already deleted this commit.